test/system/030-run.bats

```diff
@@ -1399,4 +1399,28 @@ search               | $IMAGE           |
     run_podman rm -f -t0 $cid
 }
 
+@test "podman run with mounts.conf missing" {
+    skip_if_remote "--default-mounts-file is not supported for remote clients"
+    MOUNTS_CONF=$PODMAN_TMPDIR/mounts.conf
+    run_podman run --rm --default-mounts-file=${MOUNTS_CONF} $IMAGE echo test1
+    assert "$output" = "test1" "No warning messages"
```

Comment:
Request: unique test failure messages: "No warning messages with missing mounts.conf", "...with empty mounts.conf", "with bogus mounts.conf". That makes it so much nicer for the monkey looking at test failures.